FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/main_full.c

```diff
@@ -186,6 +186,9 @@ static void prvReloadModeTestTimerCallback( TimerHandle_t xTimer );
 /* The variable into which error messages are latched. */
 static char * pcStatusMessage = "OK: No errors";
 
+int xErrorCount = 1;
```

Comment:
Can be static?

Reply:
oups already merged.. i will create a new commit for that